src/AzureIoTHubClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <AzureIoTHubClient.h>
+
+Client* AzureIoTHubClient::sslClient = NULL;
+
+AzureIoTHubClient::AzureIoTHubClient(Client& sslClient)
+{
+    AzureIoTHubClient::sslClient = &sslClient;
```

Comment:
The ```RTCZero``` library has this pattern.

Another option is to pass the ```sslClient``` in ```begin``` and rename ```AzureIoTHubClient``` to ```AzureIoTHubClientClass``` and create an instance in the library.

Thoughts?